contribs/freight/src/main/java/org/matsim/contrib/freight/carrier/CarrierPlanWriter.java

```diff
@@ -23,7 +23,9 @@
  * 
  * @author sschroeder
  *
+ * @deprecated Use {@link CarrierPlanXmlWriterV2} instead which writes the newer format
```

Comment:
just one idea triggered by this change: 
What do you think if we use the `CarrierPlanWriter` as default and call from here the `CarrierPlanXmlWriterV2`. That would have the advantage, that one could introduce a WriterV3 (internally) at one point in time, and everyone using the more general `CarrierPlanWriter`  will write out the data with the most current version. (Similar to the approach in https://github.com/matsim-org/matsim-libs/blob/master/matsim/src/main/java/org/matsim/vehicles/MatsimVehicleWriter.java).

The stuff here could be moved out to a "V1" version _or_ we could maybe use something like "MatsimCarrierPlanWriter" as "general" class and delegate from there.

I think due to the public usage of the V2 writer we cannot reduce its visibility yet, but we could deprecate it and send the user to the more general one.

Reply:
I renamed the old writer and added a new one, similar to the VehiclesWriter and other writers in MATSim.